Passes.md

```diff
@@ -0,0 +1,40 @@
+|Passes|
+|------------|
+|-correlated-propagation|
+|-scalarrepl|
+|-lowerinvoke|
+|-strip|
+|-strip-nondebug|
+|-sccp|
+|-globalopt|
+|-gvn|
+|-jump-threading|
+|-globaldce|
+|-loop-unswitch|
+|-scalarrepl-ssa|
+|-loop-reduce|
+|-break-crit-edges|
+|-loop-deletion|
+|-reassociate|
+|-lcssa|
+|-codegenprepare|
+|-memcpyopt|
+|-functionattrs|
+|-loop-idiom|
+|-lowerswitch|
+|-constmerge|
+|-loop-rotate|
+|-partial-inliner|
+|-inline|
+|-early-cse|
+|-indvars|
+|-adce|
+|-loop-simplify|
+|-instcombine|
+|-simplifycfg|
+|-dse|
+|-loop-unroll|
+|-lower-expect|
+|-tailcallelim|
+|-licm|
+|-sink|
```

Comment:
Don't we have 45 passes?

Reply:
It is all in the paper. This is stale. I will delete these two files